pkg/web/routes.go

```diff
@@ -172,13 +172,12 @@ func (s *Server) HandleBundleListFiles() http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		repoName := chi.URLParam(r, "repoName")
 		bundleID := chi.URLParam(r, "bundleID")
-		bundle := core.Bundle{
-			RepoID:           repoName,
-			BundleID:         bundleID,
-			MetaStore:        s.metadataStore(),
-			BundleDescriptor: model.BundleDescriptor{},
-		}
-		err := core.PopulateFiles(context.Background(), &bundle)
+		bundle := core.New(core.NewBDescriptor(),
```

Comment:
What is the rationale behind this type of syntactic change?

Reply:
since the `cmd/datamon/cmd` package uses `core.New` (admittedly sugar), the main idea here is consistency.

now, the `core.New` itself is something that we could view as idiomatic, a sort-of borrowed idiom from Java, etc. where constructor functions are used.  golang idioms are liable to change over time, yet in no small part because i buy into the utility of type inference, i suppose the constructor idiom is a helpful one:  we get to treat the return values of constructors as semi-loosely (i.e. duck) typed.